skypy/pipeline/tests/test_pipeline.py

```diff
@@ -44,7 +43,7 @@ def test_pipeline():
 
     pipeline = Pipeline(config)
     pipeline.execute()
-    output_filename = 'output.fits'
+    output_filename = str(tmp_path / 'output.fits')
```

Comment:
Do we need the `str` command? It is not used in the [tutorial](https://docs.pytest.org/en/6.2.x/tmpdir.html)

Reply:
"`tmp_path` is a `pathlib.Path` object" but our script takes path names as strings